Framework/Algorithms/inc/MantidAlgorithms/Stitch.h

```diff
@@ -9,6 +9,8 @@
 #include "MantidAPI/Algorithm.h"
 #include "MantidAlgorithms/DllConfig.h"
 
+using Mantid::API::MatrixWorkspace_sptr;
```

Comment:
Can we revert this change, as using namespaces in the header is strongly [discouraged](https://rules.sonarsource.com/cpp/RSPEC-1003) because anyone who includes this header will get the name at a global namespace which can cause "fun" compilation failures.

Reply:
This is not a namespace, it's just the class.

Reply:
It still pulls this class into the global namespace for all users though for this type. Whilst it might be obscure, each time this happens we undermine the point of using namespaces in the project

Reply:
Fair enough, just remove it.